testing/tests/DevExpress.animation/position.tests.js

```diff
@@ -940,20 +941,25 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
         try {
             window.innerHeight = 500;
             window.outerHeight = 200;
+            window.visualViewport = { height: 1000 };
```

Comment:
Should we set an initial object at the `finally` block?

Reply:
yeah, thx